### ISSUE TITLE
Document use of PHP's built-in server by default

### ISSUE THREAD

**Issue by webmozart** [sulu-io/sulu-docs#174]
In the Getting Started section, PHP's built-in webserver should be used by default. For getting started it's irrelevant to a user how to configure Apache or Nginx. Instead, it should be possible to set up the project as easily and quickly as possible.

**Comment by webmozart** [comment]
The following command is needed to run the web server:

```
$ app/console server:run --router app/config/router_admin.php
```

**Comment by webmozart** [comment]
See also sulu-io/sulu#2069 and sulu-io/sulu#2070.

**Comment by danrot** [comment]
Just FYI: This way the website is not accessible, and only the admin routes are available. That's not really satisfying, is it?

**Comment by webmozart** [comment]
@danrot Yes, hence: https://github.com/sulu-io/sulu-standard/issues/621 :)

**Comment by chirimoya** [comment]
FYI ... The platform.sh also needs only one front-controller. So I've added a `web/app.php` entrypoint which instantiates the kernel depending on the request url. https://github.com/sulu-io/sulu-standard/pull/601
